sources/core/objects/Human.h

```diff
@@ -42,11 +44,22 @@ class Human: public IMob
 
     void UpdateOverlays();
 
-    int GetHealth() { return health_; }
+    auto Random() const -> double;
```

Comment:
I added that accidentally, first I didn't see that I did that so I thought that was here.